renku/cli/dataset.py

```diff
@@ -118,6 +118,14 @@
       my-dataset/
         datafile
 
+You can use shell-like waldcards (e.g. *, **, ?) when specifying paths to be
+added:
+
+.. code-block:: console
+
+    $ renku dataset add my-dataset --source path/**/datafile \
```

Comment:
do the options with asterisks need to be in quotes?

Reply:
It's better to do it. If the path does not exist in the CWD then shell won't expand it and it's safe to use it without quotes.
I'll add quotes in the example.